### PR TITLE
[USBtin/Multibloc V8] - IBS integration - timeleft correction + some over modifications

### PR DESCRIPTION
**Pull request by ghost**
there was an error calculating the time remaining before charging / discharging with IBS informartion.

### CODE REVIEW

File: hardware/USBtin_MultiblocV8.cpp [hunk 1]
```diff
@@ -303,11 +305,10 @@ void USBtin_MultiblocV8::StopThread()
 	ClearingBlocList();
 }
 
-void USBtin_MultiblocV8::ManageThreadV8(bool States, bool debugmode)
+void USBtin_MultiblocV8::ManageThreadV8(bool States, unsigned int debugmode)
```
Comment (gizmocuz) [inline hardware/USBtin_MultiblocV8.cpp]: Why did you change this back to unsigned int and using/comparing it as a bool ? It is a boolean

**Comment by gizmocuz** [comment]
Thank you for the PR. Could you undo all changes and ONLY make the change for the time correction ?

**Comment by ghost** [comment]
Ok, yes I'll do that tomorrow. I don't understand why sometime when I get the code from origin reposit there is spaces and some over changes like that (tabulation, etc...), also the encoding change, from UNIX(LF) to WIN(CR+LF) so all the file is modified when I do a winmerge between domoticz reposit and mine :-D , Is there a mechanism that does automatic layout on the CI / CD side ?

**Comment by gizmocuz** [comment]
@Reivax851 , This CR/LF should be handled automatically. Maybe it is your editor ? I use Visual Studio

**Comment by ghost** [comment]
In addition, I conditioned some logs + change cast float to int on the 2 values Soc and Soh.  I compiled and tested directly on my server, everything works fine :-)

**Comment by gizmocuz** [comment]
Thank you for this PR and time again!
